metadata.json

```diff
@@ -45,6 +53,6 @@
     }
   ],
   "tags": [
-    "fail2ban"
+    "fail2ban",
```

Comment:
@phathocker You need to remove the `,` or this will fail during `metadata-json-lint`.